source/_components/hlk_sw16.markdown

```diff
@@ -0,0 +1,117 @@
+---
+layout: page
+title: "HLK-SW16 Relay Module"
+description: "Instructions on how to integrate HLK-SW16 relay into Home Assistant."
+date: 2018-10-29 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hlktech.jpg
+ha_category: DIY
+ha_release: 0.XX
+ha_iot_class: "Local Push"
+---
+
+The [HLK-SW16](http://www.hlktech.net/product_detail.php?ProId=48) by [Hi-Link](http://www.hlktech.net/) is a simple networkable 16 port relay device.
+
+To enable it, add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+hlk_sw16:
+  relay1:
+    host: 10.225.225.53
+    switches:
+      0:
+        name: relay1-0
+      1:
+        name: relay1-1
+      2:
+        name: relay1-2
+      3:
+        name: relay1-3
+      4:
+        name: relay1-4
+      5:
+        name: relay1-5
+      6:
+        name: relay1-6
+      7:
+        name: relay1-7
+      8:
+        name: relay1-8
+      9:
+        name: relay1-9
+      a:
+        name: relay1-a
+      b:
+        name: relay1-b
+      c:
+        name: relay1-c
+      d:
+        name: relay1-d
+      e:
+        name: relay1-e
+      f:
+        name: relay1-f
+
+  relay2:
```

Comment:
The first config example should be minimal and only include required config values for a normal set up.